pkg/tests/cross-tests/tfwrite.go

```diff
@@ -38,6 +39,93 @@ func WriteHCL(out io.Writer, sch map[string]*schema.Schema, resourceType, resour
 	return err
 }
 
+type PFWriter struct {
+	out io.Writer
+}
+
+func WritePF(out io.Writer) PFWriter { return PFWriter{out} }
+
+func (w PFWriter) Provider(sch pfproviderschema.Schema, providerName string, config map[string]cty.Value) error {
+	if !cty.ObjectVal(config).IsWhollyKnown() {
+		return fmt.Errorf("WriteHCL cannot yet write unknowns")
```

Comment:
Definitely out of scope but could be very nice. It could reference some proxy resource.